docs/_src/api/api/translator.md

```diff
@@ -55,20 +55,20 @@ We currently recommend using OPUS models (see __init__() for details)
 **Example:**
 
 ```python
-|    DOCS = [
-|        Document(content="Heinz von Foerster was an Austrian American scientist combining physics and philosophy,
-|                       and widely attributed as the originator of Second-order cybernetics.")
-|    ]
-|    translator = TransformersTranslator(model_name_or_path="Helsinki-NLP/opus-mt-en-de")
-|    res = translator.translate(documents=DOCS, query=None)
```

Comment:
I've seen many of these removals. Did you remove them by hand? AFAIK the pipes are useful, we should not remove them (as you can see, they ruin the code indentation)

Reply:
Ahh sorry I did remove these by hand. However, they appear as pipes in the code blocks on our website https://haystack.deepset.ai/reference/translator and I had thought we determined it was best to remove them. Since this does not allow users to copy paste the code